src/python/pants/backend/python/dependency_inference/rules_test.py

```diff
@@ -3,192 +3,208 @@
 
```

Comment:
We now use 3 distinct `RuleRunner`s rather than 1 `TestBase`. This results in tighter rule graphs.